Netlens/src/test/java/co/cask/cdap/apps/netlens/app/NetlensAppTest.java

```diff
@@ -82,7 +83,6 @@ public void testNetlensApp() throws Exception {
       }
     } finally {
       TimeUnit.SECONDS.sleep(1);
-      RuntimeStats.clearStats("");
```

Comment:
The method has been deprecated, with no new usage provided. And the tests still work without it (I ran mvn clean package).